cli/checkpoint/lua/play.lua

```diff
@@ -54,7 +54,29 @@ local function play(positional_arguments, keyword_arguments, opts)
         log.error('Internal error: empty URI is provided')
         os.exit(1)
     end
-    local remote = netbox.new(uri, opts)
+
+    local remote = nil
+    if opts.transport ~= nil and opts.transport == "ssl" then
+        remote = netbox.connect(
+            {
+                uri = uri,
+                params = {
+                    transport = opts.transport,
+                    ssl_cert_file = opts.ssl_cert_file,
+                    ssl_key_file = opts.ssl_key_file,
+                    ssl_ca_file = opts.ssl_ca_file,
+                    ssl_ciphers = opts.ssl_ciphers
+                }
+            },
+            {
+                user = opts.user,
+                password = opts.password
+            }
+        )
+    else
+        remote = netbox.new(uri, opts)
+    end
```

Comment:
Why do we need the else case? If for Tarantool 1.x support than add a comment about it.